src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -119,6 +119,8 @@
 
   {% if extraVars.help is defined and extraVars.help%}
       <small class="form-text">{{ extraVars.help }}</small>
+    {% elseif form.vars.help is defined and form.vars.help %}
```

Comment:
@matks this one has no effect using the prestashop_ui_kit template and form_widget rendering; So reverted it back like ti was before this PR. 

However using previous template and form_rest rendering it has the double label effect when we simplify forms.
Lets keep prestashop_ui_kit template and form_widget rendering and then we are out of double labels and then we can keep these lines;